packages/manager/src/features/VPC/VPCDetail/VPCDetail.tsx

```diff
@@ -1,31 +1,203 @@
+import { VPC } from '@linode/api-v4/lib/vpcs/types';
+import { Typography } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import useTheme from '@mui/styles/useTheme';
 import * as React from 'react';
 import { useParams } from 'react-router-dom';
 
+import { Box } from 'src/components/Box';
+import { Button } from 'src/components/Button/Button';
+import { CircleProgress } from 'src/components/CircleProgress/CircleProgress';
 import { DocumentTitleSegment } from 'src/components/DocumentTitle';
+import { EntityHeader } from 'src/components/EntityHeader/EntityHeader';
+import { ErrorState } from 'src/components/ErrorState/ErrorState';
 import { LandingHeader } from 'src/components/LandingHeader';
+import { Paper } from 'src/components/Paper';
+import { useRegionsQuery } from 'src/queries/regions';
+import { useVPCQuery } from 'src/queries/vpcs';
 
-const VPCDetails = () => {
+import { VPCDeleteDialog } from '../VPCLanding/VPCDeleteDialog';
+import { VPCEditDrawer } from '../VPCLanding/VPCEditDrawer';
+
+const VPCDetail = () => {
   const { vpcId } = useParams<{ vpcId: string }>();
+  const theme = useTheme();
+  const { data: vpc, error, isLoading } = useVPCQuery(+vpcId);
+
+  const { data: regions } = useRegionsQuery();
+  const regionLabel = regions?.find((r) => r.id === vpc?.region)?.label ?? '';
```

Comment:
Can we use the region slug as the fallback value?

```suggestion
  const regionLabel = regions?.find((r) => r.id === vpc?.region)?.label ?? vpc?.region;
```